Consistence/Sniffs/Exceptions/ExceptionNameEndsWithExceptionSniff.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Consistence\Sniffs\Exceptions;
+
+use PHP_CodeSniffer_File;
+use SlevomatCodingStandard\Helpers\ClassHelper;
+use StringHelper;
+
+class ExceptionNameEndsWithExceptionSniff implements \PHP_CodeSniffer_Sniff
+{
+
+	const CODE_NOT_ENDING_WITH_EXCEPTION = 'NotEndingWithException';
+
+	/**
+	 * @return int[]
+	 */
+	public function register(): array
+	{
+		return [
+			T_CLASS,
```

Comment:
I think we should listen for `T_INTERFACE` as well and apply the same rules (where applicable) for it. Also add this in the tests please and probably a test for an abstract class as well, just to be sure.